scripts/pre_commit_linter.py

```diff
@@ -1507,7 +1507,7 @@ def _lint_js_and_ts_files(
         return
 
     python_utils.PRINT('Total js and ts files: ', num_js_and_ts_files)
-    eslint_cmd_args = [node_path, eslint_path, '--quiet']
+    eslint_cmd_args = [node_path, eslint_path, '--quiet', '--fix']
```

Comment:
Is this intentional?

Reply:
Oh please ignore! You removed this in the next commit (I was reviewing commit wise).